### ISSUE TITLE
Prometheus histogram metric without "le" field

### ISSUE THREAD

**Issue by nicolasmafraintelipost** [micrometer-metrics/micrometer#3590]
**Describe the bug**
TL;DR: OpenTelemetry Collector gives error because a prometheus histogram metric doesn't have "le" field.

At endpoint `/actuator/prometheus`, metric lettuce_command_firstresponse_seconds is exposed as:
```
# HELP lettuce_command_firstresponse_seconds Latency between command send and first response (first response received)
# TYPE lettuce_command_firstresponse_seconds histogram
lettuce_command_firstresponse_seconds{application="...",command="HELLO",local="local:any",remote="...",quantile="0.5",} 0.0
lettuce_command_firstresponse_seconds{application="...",command="HELLO",local="local:any",remote="...",quantile="0.9",} 0.0
```

My OTC-container (OpenTelemetry Collector), gives this error message:
```
Unexpected error	{"kind": "receiver", "name": "prometheus", "pipeline": "metrics", "scrape_pool": "otel-collector", "target": "http://localhost:8080/actuator/prometheus", "series": "lettuce_command_firstresponse_seconds{application=\"...\",command=\"GET\",local=\"local:any\",remote=\"...\",quantile=\"0.5\"}", "error": "'le' label on histogram metric id missing or empty"}
```

**Environment**
 - Micrometer versions: 1.9.6 and 1.10.3 (both gives error)
 - Micrometer registry: prometheus
 - OS: linux
 - Java version: 11

**To Reproduce**
How to reproduce the bug:
use an application with org.springframework.boot:spring-boot-starter-data-redis:jar:2.7.6 Redis dependency.

**Expected behavior**
Prometheus histogram metrics must have "le" field.

**Additional context**
I'm using org.springframework.boot:spring-boot-starter-data-redis:jar:2.7.6 Redis dependency.
Lettuce is a library for Redis.

**Comment by jonatan-ivanov** [comment]
To repro this, I did this:
```java
Timer.builder("lettuce.command.firstresponse")
    .publishPercentiles(.999)
    .register(registry)
    .record(Duration.ofMillis(100));
```

Got this on the Prometheus endpoint:
```
# HELP lettuce_command_firstresponse_seconds  
# TYPE lettuce_command_firstresponse_seconds summary
lettuce_command_firstresponse_seconds{application="tea-service",profiles="local",quantile="0.999",} 0.096468992
...
```

And Prometheus is able to parse this. But please notice that for me, the type of the time series is not `histogram` but `summary`:
```
TYPE lettuce_command_firstresponse_seconds summary
```

If I do this:
```java
Timer.builder("lettuce.command.firstresponse")
    .publishPercentileHistogram()
    .publishPercentiles(.999)
    .register(registry)
    .record(Duration.ofMillis(100));
```

I will get the histogram but the buckets are also present:
```
# HELP lettuce_command_firstresponse_seconds  
# TYPE lettuce_command_firstresponse_seconds histogram
lettuce_command_firstresponse_seconds{application="tea-service",profiles="local",quantile="0.999",} 0.096468992
lettuce_command_firstresponse_seconds_bucket{application="tea-service",profiles="local",le="0.001",} 0.0
...
```
Prometheus is able to parse this too.

In your prometheus output, do you also see time series named as `lettuce_command_firstresponse_seconds_bucket`?
If not, could you please give us a minimal sample app that can reproduce the problem?
If you do, I'm afraid, this is an OTel issue since Prometheus is able to parse the output.

**Comment by jtama** [comment]
maybe due to the mime type. The behaviour is not the same between plain/text and openmetrics format...

**Comment by jonatan-ivanov** [comment]
@jtama There were/are definitely issues around content negotiation and even Prometheus is not consistent to itself:
- https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/18913
- https://github.com/prometheus/common/issues/456
- https://github.com/prometheus/prometheus/issues/9430

Micrometer "just" provides the data as string in the format you ask for, it does not do any content negotiation. Additionally, the Prometheus server must be able to parse both OpenMetrics and plain/text and the OTel collector should too.

Since:
- @nicolasmafraintelipost did not come back to us with a reproducer and it seems this works as it should on Micrometer side
- If it is a content negotiation issue, this issue should be open on the component that provide the endpoint (Spring Boot in this case)
- If it is a parsing issue, on the collector side, I think this should be opened for the OTel Collector

Let me close this issue, please let us know if I misunderstood something and you want us to reopen.

**Comment by strongant** [comment]
{__name__="http_server_requests_seconds", application="IP-w2erp-service", exception="None", instance="10.70.2.233:1303", job="test-service-endpoints", method="GET", namespace="uat", outcome="SUCCESS", project="hub4", quantile="0.5", status="200", uri="/actuator/prometheus"}
--


prometheus receiver process ， throws exception
：



'le' label on histogram metric is missing or empty
